### ISSUE TITLE
Instance normalization should use standard deviation (tf.nn.moments gives variance)

### ISSUE THREAD

**Issue by RobinKa** [danieltan07/tensorflow-cycle-gan#1]
https://github.com/danieltan07/tensorflow-cycle-gan/blob/master/utils.py#L48-L52
https://www.tensorflow.org/api_docs/python/tf/nn/moments

```python
mean, std = tf.nn.moments(x, axes=(1,2), keep_dims=True)
return (x - mean)/(std + eps)
```

should be

```python
mean, var = tf.nn.moments(x, axes=(1,2), keep_dims=True)
return (x - mean)/tf.sqrt(var + eps)
```
